util/send-payload-pls.sh

```diff
@@ -1,17 +1,91 @@
 #!/bin/bash
 #
-# Script to post a payload against a local webserver at each paranoia level
+# Script to post a payload against a local webserver at each paranoia level.
 #
-# Note: Webserver has to be prepared to take desired PL as Request Header "PL"
+# Note: Webserver has to be prepared to take desired PL as Request Header "PL".
+
+# WARNING: Setting the paranoia level using a header without proper authentication and authorization
+# is extremely dangerous, and is not recommended for production.
+
 # Check the access log format at https://www.netnea.com/cms/apache-tutorial-5_extending-access-log/
+# This script assumes %{ModSecAnomalyScoreIn}e is the last access log column
```

Comment:
Does it really assume this? If it does, I think this is wrong. The last item should be the response anomaly score.

Reply:
fixed